.github/workflows/npm.yml

```diff
@@ -27,11 +27,6 @@ jobs:
           registry-url: "https://registry.npmjs.org"
           cache: "yarn"
 
-      - name: Resolve latest contracts
-        run: |
-          yarn upgrade --exact \
-            @keep-network/keep-core
-
```

Comment:
If we're removing `yarn upgrade ...` we need to add `yarn install --frozen-lockfile` in order to install dependencies.

Reply:
✔️